docs/setup-mac.md

```diff
@@ -14,33 +13,16 @@ These steps are for an install on a Mac. The Windows instructions are [here](./s
 
 ### Node
 
-First check the [`.nvmrc` file](./.nvmrc) to make sure you have the correct version of Node.js installed. You can install it using several tools - NVM, Nodenv, or ASDF
+First check the [`.nvmrc` file](./.nvmrc) to make sure you have the correct version of Node.js installed. You can install it using several tools - NVM, Nodenv
 
 #### Node Version Manager (NVM) / Nodenv
 
-To use [NVM](https://nvm.sh/), follow the install directions at <https://github.com/nvm-sh/nvm#install--update-script>. Once installed, you can run `nvm install` to install the correct version of Node.js.
-
-You can manually specify a version with
-
-```sh
-nvm install v16.14.0
-nvm use v16.14.0
-```
+To use [NVM](https://nvm.sh/), follow the install directions at <https://github.com/nvm-sh/nvm#install--update-script>. Once installed, you can run `nvm use` to activate the correct version of Node.js. If you don't have the correct version installed, nvm will warn you and tell you how to install it.
 
 If using [Nodenv](https://github.com/nodenv/nodenv) follow the instructions [here](https://github.com/nodenv/nodenv#installation) to install, then run run `nodenv install`.
 
 ***Make sure to use new terminals after completing install***
 
-#### ASDF
```

Comment:
Since we don't have a `.tool-versions` file checked in to our repo, I assume the use of ASDF is a personal preference rather than a team standard. As such, I think that documenting it here could be a little confusing to a new volunteer.

Reply:
Agreed